app/models/ability.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    @user = user
+
+    if @user
+      send(@user.role.to_sym)
```

Comment:
this is not used yet, will be implemented in https://www.pivotaltracker.com/n/projects/2362592/stories/169121082